lib/prawn/emoji/image.rb

```diff
@@ -22,7 +22,7 @@ def ==(other)
       private
 
       def codepoint
-        @codepoint ||= @unicode.codepoints.map { |c| c.to_s(16) }.join.downcase
+        @codepoint ||= @unicode.codepoints.map { |c| c.to_s(16) }.join('-').downcase
```

Comment:
@xtian Please use `%04x % c.to_s` (commit: 32d55b01ef375f5ba8a6e45422e77738439b5c15 ) instead of `c.to_s(16)`. See also implementation of master:
https://github.com/hidakatsuya/prawn-emoji/blob/master/lib/prawn/emoji/image.rb#L25